### PR TITLE
Rstudio

### PR DESCRIPTION
**Pull request by jkp85**
This need to be merged after https://github.com/3Blades/core-server/pull/44

### CODE REVIEW

File: base-notebook/Dockerfile [hunk 1]
```diff
@@ -25,18 +28,33 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
     mv tini /usr/local/bin/tini && \
     chmod +x /usr/local/bin/tini
 
-# Install Jupyter notebook
+# Configure environment
```
Comment (johngriebel) [inline base-notebook/Dockerfile]: Does installing conda here replace the Python installations that you removed?

Comment (jkp85) [reply base-notebook/Dockerfile]: I just used @jgwerner image here, because previous wasn't working.

Comment (johngriebel) [reply base-notebook/Dockerfile]: Okay. @jgwerner can you confirm what exactly this change does? I'm sure it's fine, I just want to be sure we are not making unintended behavioral changes in our notebooks.

Comment (jgwerner) [reply base-notebook/Dockerfile]: I cleaned core-server and moved most of the installs related to Jupyter and Conda to base image. We still have some refactoring to do for these Notebook servers, so I suggest we go ahead and merge and then we can add a new PR for full refactor of all notebooks. Python and installs were removed to not create conflict with Conda installs. We could add other opinionated images later that don't use Conda.

**Pull request merged**